packages/contentful-extension-scripts/template/javascript/src/index.spec.js

```diff
@@ -0,0 +1,3 @@
+it('hello world', () => {
+  expect(1 + 1).toBe(2);
```

Comment:
Can we add a test that actually touches the sample extension?

Reply:
Sure, it's actually supposed to be WIP. I'm going to add proper tests today